### ISSUE TITLE
Can't execute with Windows 10 pro

### ISSUE THREAD

**Issue by pzim-devdata** [vladimiry/ElectronMail#201]
![Capture](https://user-images.githubusercontent.com/52496172/64064290-fb488680-cbee-11e9-843d-e812c3c3e854.PNG)
Hello, I have this issue when I want to start ElectronMail on Windows 10 Pro 64 bits. Could you help me please ?
I have tried the "electron-mail-3.7.1-windows-nsis-installer" and "electron-mail-3.7.0-windows-nsis-installer" packages
Do not hesitate to contact me

**Comment by vladimiry** [comment]
What is in the log file located somewhere here `c:\Users\<user>\AppData\Roaming\electron-mail\log.log
`?

**Comment by pzim-devdata** [comment]
Thanks for your reply. There is no log file
![Capture](https://user-images.githubusercontent.com/52496172/64065008-d9073680-cbf7-11e9-99c6-65f3a429824a.PNG)

**Comment by pzim-devdata** [comment]
The error is it can't found sodium.node

**Comment by vladimiry** [comment]
> The error is it can't found sodium.node

Yes, it's clear but I wanted to see a full path the file as on the screenshot it's truncated.

Normally the `sodium.node` file is supposed to be located in `c:\Users\<user>\AppData\Local\Programs\ElectronMail\resources\app.asar.unpacked\node_modules\sodium-native\build\Release\` folder on Win 10. Can you list the files in this folder?

**Comment by vladimiry** [comment]
I just unpacked the https://github.com/vladimiry/ElectronMail/releases/download/v3.7.1/electron-mail-3.7.1-windows-nsis-installer.exe file running `7za x electron-mail-3.7.1-windows-nsis-installer.exe` command and then listed the referenced folder content running `ls ./resources/app.asar.unpacked/node_modules/sodium-native/build/Release/` command and got the following result:
```
libsodium.dll  obj  sodium.exp  sodium.ilk  sodium.lib  sodium.node
```

So this confirms that the installation package includes the `sodium.node` file. Maybe you got the file somehow removed, like antivirus-like software filtered that file out from the folder?

**Comment by pzim-devdata** [comment]
Here is an overview of the folders with an "user-only" install.

![Capture](https://user-images.githubusercontent.com/52496172/64066045-f1ca1900-cc04-11e9-87aa-c9314cd02f69.PNG)
![Capture2](https://user-images.githubusercontent.com/52496172/64066048-f393dc80-cc04-11e9-850f-9b89c16bed61.PNG)
![Capture3](https://user-images.githubusercontent.com/52496172/64066049-f4c50980-cc04-11e9-84ef-ca9a957b3db5.PNG)

This is the error when I install it with "user only" option :
![Capture4](https://user-images.githubusercontent.com/52496172/64066132-d4e21580-cc05-11e9-86c3-fc5f04cbf4a7.PNG)

**Comment by vladimiry** [comment]
Does the app work when you install the package for all the users (having `user only` checkbox unchecked)?

**Comment by pzim-devdata** [comment]
No I have this error :
![64064290-fb488680-cbee-11e9-843d-e812c3c3e854](https://user-images.githubusercontent.com/52496172/64066294-48d0ed80-cc07-11e9-8bfd-bb4ac8aa55f1.PNG)

**Comment by vladimiry** [comment]
According to [this](https://user-images.githubusercontent.com/52496172/64066045-f1ca1900-cc04-11e9-87aa-c9314cd02f69.PNG) screenshot, you managed the needed files to be listed in the folder but when you run the app you still get the same error?

**Comment by pzim-devdata** [comment]
I have installed it again, with Windows Defender completly stopped, but it's still the same error. I will try to install ElectronMail on another computer with Windows 10 and I will let you know if it's working. Perhaps tonight.

**Comment by pzim-devdata** [comment]
Yes I have the file. Perhaps it's my computer. I will see with another one.

**Comment by vladimiry** [comment]
In general, it looks like there is no need to fix something in the code of this project as the issue seems to be specific to your environment/system. Let's see if it works for you on another computer.

**Comment by pzim-devdata** [comment]
Ok I will tell you that tonight. Thanks

**Comment by pzim-devdata** [comment]
It's ok now !!!!!!!!!!!!!!!!!!!!!!!!!
 I don't know why; but I have installed Visual Studio 2013 x86 and x64 and Visual Studio 2015 x86 and x64 and there is no more problem. 
I have also completly stopped Windows Defender "Controlled Folder Access". For that I have enabled it, restarted the computer and disabled it and restarted the computer again.
Yhank you so much for your help and for your app ! ;-)
